src/replicated_secret_sharing.rs

```diff
@@ -0,0 +1,191 @@
+use std::{
+    fmt::Debug,
+    ops::{Add, Mul, Neg, Sub},
+};
+
+use crate::field::Field;
+use crate::prss::Participant;
+
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub struct ReplicatedSecretSharing<T>(T, T);
```

Comment:
does this represent a share in replicated secret sharing scheme?

Reply:
This represents one helper's data within a replicated secret sharing scheme